src/main/java/com/debatetimer/domain/customize/BellType.java

```diff
@@ -0,0 +1,21 @@
+package com.debatetimer.domain.customize;
+
+import java.util.function.Predicate;
+
+public enum BellType {
+
+    AFTER_START(time -> time >= 0),
+    BEFORE_END(time -> time >= 0),
+    AFTER_END(time -> time <= 0),
+    ;
+
+    private final Predicate<Integer> timeValidator;
```

Comment:
[제안]

IntPredicate 는 Wrapping Type을 쓰지 않아서 더 빠르다는 장점이 있습니당
```suggestion
    private final IntPredicate timeValidator;
```

Reply:
수정했습니다